### ISSUE TITLE
useMarkupRegions: some fields are empty when accessing $page from template php file.

### ISSUE THREAD

**Issue by nettings** [processwire/processwire-issues#311]
### Short description of the issue
simple page with auto-included _init.php and _main.php, $config->useRegionMarkup = true.
excerpt from _main.php (instrumented for debugging this:)
```
  <div id="content">
    <h1><?php echo $page->title; ?></h1>
    <?php var_dump($page->project_description); ?>
    <div id="description>
    </div>
  </div>
```
content of home.php (template file):
```
  <div id="description">
  <?php var_dump($page->project_description); ?>
  </div>
 
```
### Expected behavior
I would expect the var_dump to yield the same result whether called in _main.php or in the substituted regions in home.php.

### Actual behavior
The variable content when called from _main.php is correct. The field exists but is empty when called from home.php:

```
 <div id="content">
    <h1>The Office Of Alina Lupu</h1>
    string(570) "<div id=pw-edit-2 class='pw-edit pw-edit-InputfieldCKEditor' data-name=project_description data-page=1 style='position:relative'><div class=pw-edit-orig><p><strong>Hello World!</strong><img alt="" src="/site/assets/files/1/img_2043---edit---composite.800x0-is.jpg" width="800" /></p></div><div class=pw-edit-copy id=pw-editor-project_description-1 style='display:none;-webkit-user-select:text;user-select:text;' contenteditable><p><strong>Hello World!</strong><img alt="" src="/site/assets/files/1/img_2043---edit---composite.800x0-is.jpg" width="800" /></p></div></div>"
    <div id="description">
  string(570) ""
  </div>
```
I should add that strangely enough, the default field "$page->title" works in both environments.

### Setup/Environment
* ProcessWire version: git HEAD from a week ago.
* (Optional) PHP version: 7
* (Optional) MySQL version: current MariaDB from opensuse Leap 42.2 
* (Optional) Any 3rd party modules that are installed and could be related to the issue: none I'm aware of.

**Comment by nettings** [comment]
It seems that my problem is ultimately caused by front-end-editing. I suspect that whenever a DOM fragment that would be replaced by $template.php under useMarkupRegions=true is also worked on by the CKEditor's front-end editing code, we have a problem.
When I'm logged out, or when I disable front-end-editing, the DOM substitution from the $template.php file works as expected.

**Comment by nettings** [comment]
Which would make this a duplicate of #294.

**Comment by nettings** [comment]
Appears to be fixed as of 3.0.66.
Can normal users tag this as solved? If so, I can't figure out how. Editing the title for now...
